src/architecture/serialization.md

```diff
@@ -243,6 +243,33 @@ in the correspondence with [the validation rules](#segment-validation-rules).
     binary format all fields of such structure are placed in the header, its body
     is empty. So such a structure is fixed-length.
 
+#### Structure composition
+
+`encoding_struct!` macro allows to make a composition of existing structures:
```

Comment:
Nitpick: "...allows to compose existing structures" sounds a bit more natural to me.